wiki/Siwecos-Siegel/wikitext/de.wiki

```diff
@@ -7,7 +7,7 @@
 -
 #########################################################
 -->
-[[File:siwecos_siegel.png|left|150px]] Um zu sehen, wann die Scanner von Siwecos Ihre Webseite das letzte Mal überprüft haben, können Sie unser Siwecos-Siegel auf Ihrer Webseite einbinden.
+[[File:siwecos_siegel.png|left|150px|alt=Das Siwecos-Siegel zeigt Ihnen, wann Ihre Seite zuletzt geprüft wurde|link=https://siwecos.de]] Um zu sehen, wann die Scanner von Siwecos Ihre Webseite das letzte Mal überprüft haben, können Sie unser Siwecos-Siegel auf Ihrer Webseite einbinden.
```

Comment:
Wäre es nicht praktischer, hier bspw. direkt das Siegel vom letzten SIWECOS-Scan einzubinden bzw. direkt zur Prüfung zu verlinken?

Die SVG-Datei (Bild) liegt hier:
https://siegel.siwecos.de/www.siwecos.de/d.m.y.svg

Der Link wäre:
https://siwecos.de/scanned-by-siwecos/?data-siwecos=www.siwecos.de

Ebenfalls dann für die EN Version.

Reply:
Noch pfiffiger wäre, wenn da das Siegel des Kunden kommen könnte… Ich denke mal drüber nach.

Reply:
Nachtrag: Schwierig, weil wir dann erstmal im WIKI externe Bilder zulassen müssten…

Reply:
Jein. Das funktinoiert doch schon. Schau Dir mal bitte, wenn Du nicht eingeloggt bist die Seite https://siwecos.de/wiki/Siwecos-Siegel/DE?userdomain=www.lednerb.eu an.

Schaue auf die Beispielcodes, die Dir angezeigt werden.

Die wiki Seite wird verlinkt von den Ergebnissen der Scans ("So binden Sie das Siegel ein" oder soähnlich). Hier bekommt der (angemeldete) Kunde direkt seinen Code angezeigt.

Das könnte theoretisch auch mit dem Siegel so geschehen. Ist aber "Verschwendung von Lebenszeit" ;) Allenfalls eine nette Spielerei.

Reply:
> Wäre es nicht praktischer, hier bspw. direkt das Siegel vom letzten SIWECOS-Scan einzubinden bzw. direkt zur Prüfung zu verlinken?
Der Gedanke ist nicht schlecht, aber im Wiki sind erstmal grundsätzlich externe Bilder verboten. Extern sind alle Bilder, die nicht ins Wiki hochgeladen werden. 

Ich habe mich damit mal beschäftigt und folgende Möglichkeiten gefunden:

in includes/DefaultSettings.php hinzufügen:

/**
 * Whether to allow inline image pointing to other websites
 */
$wgAllowExternalImages = true;

in LocalSetttings.php hinzfügen:

$wgFileExtensions[] = 'svg';
$wgAllowTitlesInSVG = true;

https://www.mediawiki.org/wiki/Manual:Image_administration#SVG
https://www.mediawiki.org/wiki/Manual:$wgAllowExternalImages

In meinem Testwiki scheitert es am SVG-Converter. Ich habe keinen Schimmer, wie ich einen der Converter einbinden könnte.

Add $wgSVGConverter to LocalSettings.php and set the renderer you want to use.

    Available options are ImageMagick, ImagickExt, sodipodi, inkscape, batik, rsvg, and imgserv.
    For example: $wgSVGConverter = 'ImageMagick';

IMHO lohnt sich dieser ganze Aufwand nicht.